crates/iota-graphql-client/Cargo.toml

```diff
@@ -45,4 +37,4 @@ iota-graphql-client-build = { version = "0.0.1", path = "../iota-graphql-client-
 
 [features]
 default = []
-uniffi = ["dep:uniffi", "iota-types/uniffi"]
```

Comment:
Why? The crate doesn't compile independently anymore if you remove it

Reply:
What do you mean? `cargo build` works fine in iota-graphql-client and iota-sdk-types

Reply:
fixed